doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -23,6 +23,56 @@ reference these objects is not allowed.
+{{< /warning >}}
+
+The `mz_activity_log` view contains a log of the SQL statements that
+have been issued to Materialize, along with various metadata about
+them.
+
+Entries in this log may be sampled. The sampling rate is controlled by
+the session variable `statement_logging_sample_rate`, which may be set
+to any value between 0 and 1. For example, to disable statement
+logging entirely for a session, execute `SET
+statement_logging_sample_rate TO 0`. Materialize may apply a lower
+sampling rate than the one set in this variable, as described in the
+warning box above.
+
+<!-- RELATION_SPEC mz_internal.mz_activity_log -->
+| Field                     | Type                         | Meaning                                                                                                                                                                                                                                                                       |
+|---------------------------|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `execution_id`            | [`uuid`]                     | An ID that is unique for each execute statement.                                                                                                                                                                                                                              |
+| `sample_rate`             | [`double precision`]         | The actual rate at which the statement was sampled.                                                                                                                                                                                                                           |
+| `cluster_id`              | [`text`]                     | The ID of the cluster the statement execution was directed to.                                                                                                                                                                                                                |
+| `application_name`        | [`text`]                     | The value of the `application_name` session variable at execution time.                                                                                                                                                                                                       |
+| `cluster_name`            | [`text`]                     | The name of the cluster with ID `cluster_id` at execution time.                                                                                                                                                                                                               |
```

Comment:
Implementation detail, but I wish `cluster_name` was positioned immediately after `cluster_id`.

Reply:
I agree, will change in a follow-up